README.md

```diff
@@ -21,16 +21,31 @@ These are important benefits:
    source-level diff, LLpatch uses LLVM IR for diffing to abstract away all the
    specifics/details on machine architecture, types of linkers, and kernel version.
 
-2. **Slim Livepatch**: LLpatch uses 'llvm-diff' to perform the diffing and generates LLVM IR
+2. **Easy Callbacks and LLPatch Symbols**: Kernel livepatch allows four callbacks
+   before/after patching/unpatching the livepatch. LLpatch helps livepatch developer to
```

Comment:
It seems odd to say "Kernel livepatch allows four callbacks ... the livepatch." Could you reword that?

And I think it should be "LLpatch helps the livepatch developer implement".

Reply:
Yup.